test/fixtures/mock-npm.js

```diff
@@ -316,104 +316,112 @@ const loadNpmWithRegistry = async (t, opts) => {
     )
   }
 
-  const assert = { fileShouldExist, fileShouldNotExist, packageVersionMatches }
-
-  return { registry, assert, ...mock }
-}
-
-/** breaks down a spec "abbrev@1.1.1" into different parts for mocking */
-function dep (spec, opt) {
-  const [name, version = '1.0.0'] = spec.split('@')
-  const lockPath = !opt.hoist && opt.parent ? `${opt.parent}/` : ''
-  const { definition } = opt
-
-  const depsMap = definition ? Object.entries(definition).map(([s, o]) => {
-    return dep(s, { ...o, parent: name })
-  }) : []
-  const dependencies = Object.assign({}, ...depsMap.map(d => d.asDependency))
-
-  const asPackageJSON = JSON.stringify({
-    name, version, ...(Object.keys(dependencies).length ? { dependencies } : {}),
-  })
-
-  const asDependency = {
-    [name]: version,
+  const packageInstalled = (target) => {
+    const spec = path.basename(target)
+    const dirname = path.dirname(target)
+    const [name, version = '1.0.0'] = spec.split('@')
+    assert.fileShouldNotExist(`${dirname}/${name}/${name}@${version}.txt`)
```

Comment:
We should probably stick to using `t` for assertions.

Reply:
This function uses `t` under the hood. I can make this `assert.packageInstalled('workspace-b/node_modules/abbrev@1.1.1').map(v => t.equal(...v))` pretty easily, does that make sense?